scripts/bulk-password-change.js

```diff
@@ -44,24 +45,20 @@ const options = {
 };
 
 const execute = async () => {
+  const extraWarning = '\nThey will ALL have the same password "Secret_1"\n';
+  const input = await areYouSure('ALL', extraWarning);
+  if (input.toLowerCase() !== 'y') {
+    process.exit(0);
+  }
   let users = [];
   try {
     users = await rpn.get(options);
   } catch (e) {
     console.log('An error while getting the list of users - ', e.message);
   }
-  
+
   users.forEach(async (user) => {
-    const postOptions = Object.assign({}, options);
-    postOptions.body = {
-      'password': 'Secret_1'
-    };
-    postOptions.uri = `${options.uri}/${user.username}`;
-    try {
-      await rpn.post(postOptions);
-    } catch (e) {
-      console.log('An error while updating the password - ', e.message);
-    }
+    await changeUserPass(user.username, 'Secret_1',  options);
```

Comment:
I don't understand the use case for this script. When would someone want to update all passwords to the same insecure one? I think we should deprecate this use case and not recommend it for anyone.

It's convenient to have a "reset all" script, but the passwords should be randomly generated (as in the other script).

Let's just delete this script to be safe. Then if this is a useful feature still, work to implement an `--all` flag in the other script so people have to type something extra to do the more destructive option.

Reply:
There [was a use case](https://github.com/medic/cht-core/pull/7080#pullrequestreview-648719995) that was needed for a QA to test back when. but fair! I'll un-refactor the changes in [9b2d197](https://github.com/medic/cht-core/pull/8874/commits/9b2d1976b5faa8c7f6531d1fc2c15c66fe6d0107), delete the dangerous one and resubmit

Reply:
fixed in 32adfe536!

also @billwambua had some changes he wanted to handle quoted strings which he added with a `String.prototype.splitCSV()` call.